frontend/packages/client/src/App.sass

```diff
@@ -20,6 +20,7 @@ $title-family: $family-secondary
 $body-color: black
 $body-background-color: white
 $primary: $yellow
+$spacing-values: ("0": 0, "1": 0.25rem, "2": 0.5rem, "3": 0.75rem, "4": 1rem, "5": 1.5rem, "6": 3rem, "7": 4.5rem,"8": 6rem, "9": 9rem, "auto": auto) !default
 
```

Comment:
This will enable us do use classNames with `p-7`, `p-8` or `p-9` instead of using the style prop of the component